app.py

```diff
@@ -100,178 +101,37 @@ def _project_list_query():
 import app_callbacks
 
 # CREATE APP OBJECT
-load_figure_template(["sandstone", "minty"])
+load_figure_template(["sandstone", "minty", "slate"])
+
+# stylesheet with the .dbc class
+dbc_css = "https://cdn.jsdelivr.net/gh/AnnMarieW/dash-bootstrap-templates/dbc.min.css"
+
```

Comment:
it'd be nice to have some comments here about this, e.g. "creating a custom theme for site called 'custom_dark' in the plt_io.templates dictionary"